pkg/client/client.go

```diff
@@ -0,0 +1,209 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/openshift/cluster-network-operator/pkg/util/k8s"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+	kinformer "k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog/v2"
+
+	osoperclient "github.com/openshift/client-go/operator/clientset/versioned"
+	osoperinformer "github.com/openshift/client-go/operator/informers/externalversions"
+	operatorv1helpers "github.com/openshift/library-go/pkg/operator/v1helpers"
+
+	configv1 "github.com/openshift/api/config/v1"
+	operv1 "github.com/openshift/api/operator/v1"
+	netopv1 "github.com/openshift/cluster-network-operator/pkg/apis/network/v1"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const defaultResyncPeriod = 5 * time.Minute
```

Comment:
is this the common value on other operators? I take it's not usually parameterizable if so...

Reply:
Looking around, many operators are between 1 minute and 2 hours. We've always kept to 5 minutes, which seems fine.